.github/workflows/python_bindings.yml

```diff
@@ -251,13 +269,84 @@ jobs:
       run: |
           set -x
           python -m pip install --upgrade pip
-          pip install requests packaging
+          # Note: if we choose to do ALL or nothing, remove  setuptools wheel twine (not needed unless trying to upload within this step)
+          pip install requests packaging setuptools wheel twine
           bindings_v=$(python ./python/find_pypi_tag.py --current)
           pip install -i https://test.pypi.org/simple/ openstudio==$bindings_v
+          mkdir wheel
 
     - name: Test the bindings
       shell: python
       run: |
           import openstudio
           m = openstudio.model.exampleModel()
           print(m)
+
+    - name: Download the specific wheel
+      # if: contains(github.ref, 'refs/tags')
+      uses: actions/download-artifact@v2
+      with:
+        path: ./wheel
+        name: openstudio-${{ matrix.name }}
+
+    - name: Extract the tar.gz, then upload
+      # if: contains(github.ref, 'refs/tags')
+      working-directory: ./wheel
+      env:
+        TWINE_USERNAME: __token__
+        TWINE_REPOSITORY: pypi
+        # Go to repo on github > Settings > Secrets and add it
+        TWINE_PASSWORD: ${{ secrets.PYPI_TOKEN }}
+      shell: bash
+      run: |
+        tar -xzvf "openstudio-${{ matrix.name }}.tar.gz";
+        ls -R
+        python -m twine upload --skip-existing dist/*
+
+  upload_python_bindings_to_pypi:
+    # if: contains(github.ref, 'refs/tags')
+    needs: test_python_bindings
+    name: Upload to PyPi
+    runs-on: ubuntu-20.04
+    steps:
```

Comment:
This job is run if the entire test job works. And we try to upload everything in one go. (Obviously in this case since we already did above, the upload would be skipped...)